feature-libs/cart/base/components/cart-shared/cart-item-list/cart-item-list.component.ts

```diff
@@ -107,7 +108,9 @@ export class CartItemListComponent implements OnInit, OnDestroy {
   protected getInputsFromContext(): Subscription | undefined {
     return this.outlet?.context$.subscribe((context) => {
       if (context.readonly !== undefined) {
+        this._forceReRender = this.readonly !== context.readonly;
         this.readonly = context.readonly;
+        this.cd.markForCheck();
       }
       if (context.hasHeader !== undefined) {
```

Comment:
Why not `if(context.hasHeader ) {...}`

Reply:
@Uli-Tiger, you could use a short version. See above.

Reply:
@Larisa-Staroverova: agree - but as you can see it is existing code of an reuse component, which i want to avoid to refactor, so we keep a clean diff and it is clear what gets added/changed.